events/Event.h

```diff
@@ -227,39 +219,64 @@ class Event<void()> {
 
 public:
     /** Create an event
-     *  @see Event::Event
```

Comment:
Removed these as they did not link properly and I had moved all parameter documentation to each function. I can add it back in since the referenced function does have some additional description.